lib/SwiftDemangle/CMakeLists.txt

```diff
@@ -3,7 +3,12 @@ add_swift_library(swiftDemangle SHARED
   MangleHack.cpp
   LINK_LIBRARIES swiftDemangling)
 if(NOT APPLE)
-  target_link_libraries(swiftDemangle PRIVATE bsd)
```

Comment:
[This is all one command](https://cmake.org/cmake/help/v3.3/command/target_link_libraries.html#libraries-for-a-target-and-or-its-dependents): link the build product `swiftDemangle`, `PRIVATE`ly, to `bsd`. You can just change the condition.

Reply:
Thank you Jordan. I still have hard time reading CMake ;)